### ISSUE TITLE
How to create project from repo that doesn't have a .sln file??

### ISSUE THREAD

**Issue by drankinatty** [MicrosoftDocs/visualstudio-docs#9771]
I have had a github account for years and usually just use `make` to build the files. Even for the windows desktop repositories, I don't include a `.sln` file. So I thought I would try the VS clone a repository feature. I connected to github, cloned the repo and it cloned fine into the solution explorer windows -- but now it is just sitting there.

![vs-git-clone](https://github.com/MicrosoftDocs/visualstudio-docs/assets/9997638/cd131ba6-f2d6-4a50-ba17-2c1a14c26649)

It would be nice for this page to mention or describe how to create a project from the cloned repository if there is no `.sln` project file found. (or at least provide a link to the page that does)

At least a quick note about `File -> New -> Project` and choosing the repository as the source. I finally found it, but for someone new working with git in VS for the first time, an extra sentence in the *Clone a repo in Visual Studio* page would be helpful.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 47681749-a303-6964-5aaf-50669805b780
* Version Independent ID: 47681749-a303-6964-5aaf-50669805b780
* Content: [Clone a repo in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/version-control/git-clone-repository?view=vs-2022)
* Content Source: [docs/version-control/git-clone-repository.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/version-control/git-clone-repository.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**

**Comment by WilliamAntonRohm** [comment]
@drankinatty -- David, thank you for your thoughtful feedback.  Please consider these resources:
- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- Check the [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio). 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@ghogen -- Gordon, please look into this issue.

**Comment by ghogen** [comment]
@drankinatty 
I'm looking again at this, and I guess I need to ask for more details about what you mean about using FIle > New > Project and then choosing the folder as a source.  When you use File > New > Project, the first screen gives you a choice of project templates, so are you choosing a project type there and then advancing to the next screen? The next screen asks for the name and location of the project. Are you saying that you're choosing the location of the folder in the cloned repo at that point? I'm interested in this scenario, but I'm having trouble following how you solved this situation.

**Comment by drankinatty** [comment]
I apologize, but I think you misunderstood the sequence of operations giving rise to this question. The problem occurs when you clone a repo as the source for a project that does not contain a `.sln` file. That is the first operation, the lack of documentation pertains to then choosing `File -> New -> Project` and choosing the cloned repository as the source. So the sequence is:

1. Use the visual studio clone a repository feature for a repo that does not contain a `.sln` file. (all files appear in the solution explorer -- but you cannot make use of them because there is no association to build created by cloning)
2. (solution) `File -> New -> Project` and choose the cloned repository as the source invokes the process that prepares the project to build.

The problem is there is nothing in the documentation about cloning a repository that explains you then must do step 2. I had to simply hunt and peck thinking about what would be a logical way to get visual studio to configure the repository it just cloned for me into something I could build. There needs to be some documentation in the clone a repository information that describes the steps needed to make use of what you just cloned. Otherwise you are just left staring at a bunch of files in solution explorer that visual studio has no clue what to do with.

Let me know if that fills in the gaps, otherwise, I'm happy to try again.

**Comment by ghogen** [comment]
@drankinatty
I have repeated step 1, I opened a repo with project files, but no solution files. It opens in Solution Explorer.
Next, I choose File >New Project from the main menu. (I tried seeing if there was another UI gesture you might mean, by right-clicking on the Solution Explorer, but did not get anything that resembles File > New > Project).
After that, I see the screen "Create a new project" with options for project templates. I need a little more detail on what you mean by "choose the cloned repository as the source" - how is this accomplished in Visual Studio?
